naughtystrings/naughtystrings.go

```diff
@@ -0,0 +1,37 @@
+// Package naughtystrings is a collection of strings that have a high probability of causing issues when used as user input.
+package naughtystrings
+
+import (
+	"encoding/json"
+	"os"
+	"path"
+	"runtime"
+)
+
+// Base64Encoded returns the strings encoded in base 64.
+func Base64Encoded() []string {
+	return strings("blns.base64.json")
+}
+
+// Unencoded returns the strings.
+func Unencoded() []string {
+	return strings("blns.json")
+}
+
+func strings(file string) []string {
+	var _, here, _, _ = runtime.Caller(0)
+
+	there, err := os.Open(path.Join(path.Dir(path.Dir(here)), file))
```

Comment:
Now when you mention it, there should be warning in docs that this library will work only on machines with sources of this repo. Building on one machine and moving resulting binary to another will not work.